ts/app.ts

```diff
@@ -1,9 +1,5 @@
-import {
-  ComponentAnnotation as Component,
-  ViewAnnotation as View,
-  bootstrap,
-  For
-} from 'angular2/angular2';
+/// <reference path="../typings/angular2/angular2.d.ts" />
+import {Component, View, bootstrap, For} from 'angular2/angular2';
```

Comment:
Why does this work? Shouldn't it still be `ComponentAnnotation as Component` / `ViewAnnotation as View`?